From 705a6f8000c9a2e64bd94fad49d1c20fefcbda2f Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Thu, 10 Jul 2014 13:55:41 +0800
Subject: [PATCH 342/602] ENGR00321923 Sensor: fix light sensor not work issue

Change to use new device node path for isl29023 light sensor.
For sabresd and sabreauto, the node path is different.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 218e122..f791224 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -38,10 +38,16 @@
 
 /sys/devices/platform/mxsdhci.2/mmc_host/mmc0/mmc0:000* boot_bus_config 0660 root root
 /sys/devices/platform/mxsdhci.2/mmc_host/mmc0/mmc0:000* boot_config 0660 root root
-/sys/devices/platform/imx-i2c.2/i2c-2/2-0044 mode 0660 system system
-/sys/devices/platform/imx-i2c.2/i2c-2/2-0044 lux 0660 system system
-/sys/devices/platform/imx-i2c.2/i2c-2/2-0044 int_lt_lux 0660 system system
-/sys/devices/platform/imx-i2c.2/i2c-2/2-0044 int_ht_lux 0660 system system
+# for sabresd
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-2/2-0044 mode 0660 system system
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-2/2-0044 lux 0660 system system
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-2/2-0044 int_lt_lux 0660 system system
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-2/2-0044 int_ht_lux 0660 system system
+# for sabreauto
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-1/1-0044 mode 0660 system system
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-1/1-0044 lux 0660 system system
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-1/1-0044 int_lt_lux 0660 system system
+/sys/devices/soc0/soc.1/2100000.aips-bus/21a8000.i2c/i2c-1/1-0044 int_ht_lux 0660 system system
 /sys/devices/platform/mxc_sdc_fb.0/graphics/fb*	fsl_disp_property 0660 system graphics
 /sys/devices/platform/mxc_sdc_fb.1/graphics/fb*	fsl_disp_property 0660 system graphics
 /sys/devices/platform/mxc_sdc_fb.0/graphics/fb*	mode 0660 system graphics
-- 
1.8.0

